dev/TabView/TabView.idl

```diff
@@ -7,6 +7,16 @@ enum TabViewWidthMode
 {
     Equal = 0,
     SizeToContent = 1,
+    Compact = 2,
+};
+
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+enum TabViewCloseButtonOverlayMode
```

Comment:
Has this gone through API review? @ranjeshj

Reply:
This particular API hasn't gone through formal API review, not yet.

Reply:
Please use [WUXC_VERSION_PREVIEW] instead of [WUXC_VERSION_MUXONLY] to unblock the PR before the API review is complete. Once the API review is completed, we can make any necessary changes and switch it. Doing this will make it public in the WinUI2.x pre-release package but not the stable package.

Reply:
I did not knew that. Changed it now :)